docs/news/posts/2024-12-13-core-plugins.md

```diff
@@ -0,0 +1,140 @@
+---
+blogpost: true
+category: Blog
+tags: plugins
+author: Julian Geiger
+date: 2024-12-13
+---
+
+# Non research-domain specific plugins to extend core functionality
+
+Dear users, we're happy to announce that we have recently crossed the 100-plugin milestone of registered plugin packages on the [AiiDA plugin registry](https://aiidateam.github.io/aiida-registry/).
+Among those are classics like [`aiida-quantumespresso`](https://github.com/aiidateam/aiida-quantumespresso/) and [`aiida-vasp`](https://github.com/aiidateam/aiida-vasp/), which support the electronic structure codes we all know and love.
+However, did you know that there are various non research-domain specific plugins that AiiDA developers have created over the years, which can extend AiiDA's core functionality?
+In today's post, we would like to give you a short introduction to some of these plugins, and show you when you can incorporate them into your work, to make sure you'll get the most out of your AiiDA journey![^1]
+
+## Simplifying code execution and workflow creation
+
+### [`aiida-workgraph`](https://github.com/aiidateam/aiida-workgraph/)
+
+We've been making quite a bit of fuzz about this one in recent weeks already, as we're targeting a first, stable release for early next year (fingers crossed :crossed_fingers:).
+The `aiida-workgraph` provides an alternative way to write workflows, in addition to (but not replacing) the classical `WorkChain`.
+It comes with all the typical AiiDA features, like provenance tracking, checkpointing, caching, and remote execution, but provides a simpler syntax, and allows you to write workflows more interactively.
+This can be achieved by adding graph nodes[^2] (or `Tasks`, as we call them) and links, in equivalence to how one would typically build-up a graph.
+In this way, `aiida-workgraph` makes it simpler to quickly connect existing AiiDA components (`CalcJobs`, `WorkChains`, or other `WorkGraphs`), and even allows visualizing the flow of data via the different processes before the workflow is executed (through an interactive web GUI).
+As we're currently stabilizing the API, any feedback is valuable and helps us to polish the tool as best as we can before the release!
+You can read more about it on its [RTD page](https://aiida-workgraph.readthedocs.io/en/latest/), and find example `WorkGraphs` in [this repository](https://github.com/superstar54/workgraph-collections).
+
+### [`aiida-shell`](https://github.com/sphuber/aiida-shell/)
```

Comment:
Same as above, link to docs may be better?

Reply:
Same comment as above.